app/helpers/mailer_helper.rb

```diff
@@ -56,6 +46,14 @@ def add_reason(text, reason)
     end
   end
 
+  def proposal_attributes(proposal)
```

Comment:
good point - we already have a `fields_for_display` method on each client data and that method is delegated from `proposal` (we use it in the web views) so we can take advantage of that here